### PR TITLE
Removal of libnapc and libnapc-nightly

### PR DESCRIPTION
**Pull request by sz-coder**
Hello,

I'll shortly make a new pull request because the URL/home of my project changed.

I want the library to be removed first because I don't want to host the current versions that are available on the registry today (version v1.4.1 resp. v26).

### CODE REVIEW

**Comment by per1234** [comment]
Hi @sz-coder. Although anyone is welcome to submit a library, removals or other changes to registry entries may only be done with the authorization of a maintainer of the library. Unfortunately, I was unable to verify that this request was authorized.

Please provide proof that this request is authorized.

A request is considered authorized if any of the following conditions is met:

- User submitting the request owns the library repository.
- User submitting the request is listed in the repository's [`CODEOWNERS` file](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners).
- A reference is provided to a discussion in the library's repository where authorization for the request was given by a user who meets one of the following criteria:
  - Tagged as "**Owner**", "**Member**", or "**Collaborator**"
  - Listed in the repository's `CODEOWNERS` file.

You are welcome to suggest some other method of confirming authorization if something else is more convenient.

Thanks!

**Comment by per1234** [comment]
Thanks, and I apologize for the inconvenience.

I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]
Done:
```
2022/08/31 16:16:09 Loaded 5123 libraries from DB
Removing libnapc
Removing libnapc-nightly
Success!
```

**Pull request merged**
